lib/create-contentful-api.ts

```diff
@@ -512,234 +511,49 @@ export default function createContentfulApi<OptionType extends ChainOptions>(
   }
 
   async function getEntry<Fields extends FieldsType>(id: string, query: EntryQueries = {}) {
-    return makeGetEntry<Fields>(id, query, options) as unknown
+    return (await makeGetEntry<Fields>(id, query, options)) as unknown
```

Comment:
Is there a reason why we expose the result explicitly as `unknown` instead of typing `makeGetEntry` return type and all of the inner methods? Same for `getEntries`.

Reply:
I removed it to test and it still works. Let’s see if I run into any issues. It shouldn’t be needed.